shortcode.php

```diff
@@ -394,9 +395,16 @@ function geoip_detect2_shortcode_show_if($atts, $content = null, $shortcodeName
 
     // Country
     if ($atts_array['country'] != null) {
-        if ($info->country->name && $atts_array['country'] != $info->country->name
-            && $info->country->isoCode && $atts_array['country'] != $info->country->isoCode) {
-            $criteria_test_flag = false;
+        $temp_attribute_values = explode(',', $atts_array['country']);
+
+        foreach ($temp_attribute_values as &$value) {
+            $value = trim($value);
+            unset($value);
+        }
```

Comment:
Great, you found out how it works!
These four lines are exactly what `array_walk($temp_attribute_values, 'trim')` would do.

In PHP, I try to avoid using the foreach by reference, because it somehow changes the way the array is built. If you do a `var_dump($temp_attribute_values)` you see that it is now an array of references, and normally this works as well, but I remotely remember some problem where it didn't.